src/app/reports/invoice/invoice.html

```diff
@@ -16,13 +16,13 @@ <h3 class="panel-title">Delivery Rounds</h3>
       </tr>
     </thead>
     <tbody>
-      <tr ng-repeat="round in reportsCtrl.deliveryRounds">
+      <tr ng-repeat="round in reportsAllCtrl.deliveryRounds">
```

Comment:
@freefony noted, like i said, i just moved this file, and i think we already have a ticket for paginating and optimising this invoice table Haykel created.